plugins/cockroachdb_logs.yaml

```diff
@@ -0,0 +1,260 @@
+    type: bool
+    default: true
+  - name: dev_log_path
+    description: The absolute path to the CockroachDB Dev Logs
+    type: "[]string"
+    default:
+      - "/var/log/cockroach-data/logs/cockroach.log"
+  - name: enable_error_log
+    description: Enable to collect stderr logs.
+    type: bool
+    default: true
+  - name: error_log_path
+    description: The absolute path to the CockroachDB stderr logs
+    type: "[]string"
+    default:
+      - "/var/log/cockroach-data/logs/cockroach-stderr.log"
+  - name: enable_sql_schema_log
+    description: Enable to collect sql schema logs.
+    type: bool
+    default: true
+  - name: sql_schema_log_path
+    description: The absolute path to the CockroachDB sql schema logs
+    type: "[]string"
+    default:
+      - "/var/log/cockroach-data/logs/cockroach-sql-schema.log"
+  - name: enable_telemetry_log
+    description: Enable to collect telemetry logs.
+    type: bool
+    default: true
+  - name: telemetry_log_path
+    description: The absolute path to the CockroachDB telemetry logs
+    type: "[]string"
+    default:
+      - "/var/log/cockroach-data/logs/cockroach-telemetry.log"
+  - name: enable_kv_distribution_log
+    description: Enable to collect kv distribution logs.
+    type: bool
+    default: true
+  - name: kv_distribution_log_path
+    description: The absolute path to the CockroachDB kv distribution logs
+    type: "[]string"
+    default:
+      - "/var/log/cockroach-data/logs/cockroach-kv-distribution.log"
+  - name: enable_pebble_log
+    description: Enable to collect cockroachdb pebble logs.
+    type: bool
+    default: true
+  - name: pebble_log_path
+    description: The absolute path to the CockroachDB pebble logs
+    type: "[]string"
+    default:
+      - "/var/log/cockroach-data/logs/cockroach-pebble.log"
+  - name: start_at
+    description: At startup, where to start reading logs from the file ('beginning' or 'end')
+    type: string
+    supported:
+      - beginning
+      - end
+    default: beginning
+  - name: timezone
+    description: Timezone to use when parsing the timestamp
+    type: timezone
+    default: UTC
+template: |
+  receivers:
+    {{ if .enable_health_log }}
+    filelog/cockroachdb_health_logs:
+      include:
+        {{ range $fp := .health_log_path }}
+        - '{{ $fp }}'
+        {{ end }}
+      multiline:
+        line_start_pattern: '[IWEF]\d{6}\s\d{2}:\d{2}:\d{2}\.\d{6}\s'
+      start_at: {{ .start_at }}
+      attributes:
+        log_type: 'cockroach.health'
+      operators:
+        - type: regex_parser
+          regex: '^(?P<Type>[IWEF])(?P<timestamp>\d{6}\s+\d{2}:\d{2}:\d{2}\.\d{6})\s+(?P<GoID>\d+)\s*(?:(?P<Channel>\d+)@)?(?P<Location>\S+)\s+(?:)⋮?\s+(?P<Node>\[[^\]]+\])\s+(?:(?P<Counter>\d+\s+)?)(?P<Message>.*)'
+          timestamp:
+            parse_from: attributes.timestamp
+            layout: '%y%m%d %H:%M:%S.%f'
+            location: {{ .timezone }}
+          severity: 
```

Comment:
I think the severities need a mapping. They are showing as unspecified for me. An example is [here](https://github.com/observIQ/observiq-otel-collector/blob/release/v1.25.0/plugins/apache_http_logs.yaml#L74-L81)